addons/mass_mailing/models/mailing.py

```diff
@@ -162,6 +162,7 @@ def _get_default_mail_server_id(self):
     bounced_ratio = fields.Integer(compute="_compute_statistics", string='Bounced Ratio')
     clicks_ratio = fields.Integer(compute="_compute_clicks_ratio", string="Number of Clicks")
     next_departure = fields.Datetime(compute="_compute_next_departure", string='Scheduled date')
+    is_body_empty = fields.Boolean(compute="_compute_is_body_empty", help='Technical field used to determine if the mail body is empty')
```

Comment:
Fields should be classified to keep a logical order when reading them. Probably to move under 'body_html' .